Sources/App/Model/Event.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Event.swift
+//  
+//
+//  Created by Joe Williams on 10/04/2022.
+//
+
+import Foundation
+import Fluent
+import Vapor
+
+final class Event: Model, Content, Codable {
+    
+    typealias IDValue = UUID
+    
+    static var schema: String {
+        return "events"
+    }
+    
+    @ID(key: .id)
+    var id: UUID?
+    
+    @Field(key: "name")
+    var name: String
+    
+    @Field(key: "event_date")
+    var date: Date
+    
+    @Field(key: "location")
+    var location: String
+    
+    @Children(for: \.$event)
+    var presentations: [Presentation]
+    
+    init() { }
+    
+    class Migrations: AsyncMigration {
```

Comment:
Might be worth putting migrations into their own files in a Migrations directory?